horovod/common/gradient_aggregation.py

```diff
@@ -0,0 +1,266 @@
+import tensorflow as tf
```

Comment:
Since `tensorflow` is imported here, can we move this to `horovod/tensorflow`?

Reply:
Moved to `horovod/tensorflow/gradient_aggregation.py`.